app_grim_reefer/boards/arm/nucleo_f446re/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# STM32F446RE Nucleo board configuration
```

Comment:
Wipe the nucleo_f446re board folder. Defined in the zephyr repo and we could have a .overlay file for it later if we want to test on a breakout

Reply:
fixed in [6cf8032](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/6cf803229d38690de83d6db332228645b0495534)